app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -274,6 +274,11 @@ def validate_channel(self, value):
 
 
 class NimbusDocumentationLinkSerializer(serializers.ModelSerializer):
+    def validate_title(self, value):
+        if value != NimbusExperiment.DocumentationLink:
```

Comment:
I think this is a problem spot... The condition is that `value` is exactly equal to a reference to the `DocumentationLink` class. But *think* what you mean is that it should be one of the `.choices`

Reply:
But, I also think you can just remove this, because I *think* that [having added `choices` to the model](https://github.com/mozilla/experimenter/pull/4445/files#diff-36bd393fce70ef5fc7ad821bc3c232a4b00882f000583f222fc3041f735d3bdbR208) should take care of this. All the tests pass if I comment out the whole `validate_title` method.

Reply:
Ahh yup, good catch.